### PR TITLE
updated getting-help.md to point to discord

### PR DESCRIPTION
**Pull request by barshirtcliff**

### CODE REVIEW

**Comment by armanbilge** [comment]
Scala 3 build failure here seems to have been introduced in dcc7a9f559fdaa42ba1d39138d9a00092f4ea7c6 and I don't know why.

**Comment by barshirtcliff** [comment]
> Scala 3 build failure here seems to have been introduced in [dcc7a9f](https://github.com/http4s/http4s/commit/dcc7a9f559fdaa42ba1d39138d9a00092f4ea7c6) and I don't know why.

Wow I wish I could hop over and try to help uncover the root cause.  I can't though.  Not today.

**Comment by armanbilge** [comment]
No worries, I'm on it 😆

**Review by armanbilge — APPROVED** [review]

File: website/src/hugo/content/getting-help.md [hunk 1]
```diff
@@ -2,14 +2,13 @@
...
-This channel is open to user questions of all levels of experience.
-More general questions about Cats, FS2, HTTP, Scala, etc. are always
-welcome, but may politely be redirected to a more specific forum.  We
-are less concerned about staying strictly on topic than helping
-people.  If in doubt, ask.
+The discussion forum for http4s is [http4s/http4s on the Typelevel
+Discord][Discord].  This Discord is open to user questions of all
+levels of experience.  There you will also find channels for questions
+and discussion about Cats, FS2, skunk, and much more.  Please try to
+keep your questions and contributions in these channels topical.
```
Comment (bplommer) [inline website/src/hugo/content/getting-help.md:11]: This feels mildly unwelcoming to me - is it addressing a real problem?

Comment (armanbilge) [reply website/src/hugo/content/getting-help.md:11]: That's a good point, let's just drop it?
```suggestion
and discussion about Cats, FS2, skunk, and much more!
```

Comment (armanbilge) [reply website/src/hugo/content/getting-help.md:11]: If we want to have a sentence, maybe it should be about Scala CoC and everyone is welcome :)

**Pull request merged**